libs/entry-components/styles/modules/vendors/redactor/_tables-generator.scss

```diff
@@ -0,0 +1,29 @@
+@use 'sass:map';
+@use '../../../../../scss-foundation/src/modules/lists/row-coloring' as list;
+
+$material-odd-background: #F0F0F0;
+
+@mixin generate-from($theme) {
+	$theming-table: map.get($theme, 'tables', 'rows', 'odd-even-background');
+
+	.enigmatry-redactor-content table {
+		@if $theming-table {
+			tr {
+				@include list.odd-row-coloring(map.get($theming-table, 'odd'));
+				@include list.even-row-coloring(map.get($theming-table, 'even'));
+			}
+		}
+		@else {
+			tr {
+				@include list.even-row-coloring($material-odd-background);
```

Comment:
if is this for even row then var name should be material-even-background

Reply:
Now we get values from Material, so no more hardcoding